lib/resin-lint.ts

```diff
@@ -150,7 +146,7 @@ const lintTsFiles = function(
 				return 1;
 			}
 		}
-		linter.lint(file, source, config as IConfigurationFile);
+		linter.lint(file, source, config as tslint.Configuration.IConfigurationFile);
```

Comment:
Wouldn't it be better to run `linter.lint()` _before_ `prettier.format()`? Because, with the `--fix` option, `linter.lint()` could modify the source code (right?), in which case a subsequent `prettier.format()` call could further modify the source code. In other words, it looks like running `resin-lint --typescript --fix` could result in an "unstable" state where running `resin-lint --typescript --fix` a second time would result in further source code changes.

Reply:
I've moved this comment to another `node-resin-lint` PR: https://github.com/balena-io/node-resin-lint/pull/45#discussion_r374736300